src/mn_setup.c

```diff
@@ -2061,6 +2061,8 @@ static setup_menu_t auto_settings1[] = {
     {"Color Keyed Doors", S_CHOICE, H_X, M_SPC, {"map_keyed_door"},
      .strings_id = str_automap_keyed_door},
 
+    {"Aspect Ratio Correction", S_ONOFF, H_X, M_SPC, {"correctautomapaspect"}},
```

Comment:
How about "Square aspect ratio"?

Reply:
Sounds good! I agree with @rfomin that the terminology is a bit weird, since we're trying to _counteract_ the existing aspect ratio correction so this one part looks right. This name is probably better